docs/quickstart/send-report-ui.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: "Send observability report"
+---
+
+import InstallCLI from "../snippets/install-cli.mdx";
+import InstallDbtPackage from "../snippets/install-dbt-package.mdx";
+
+Distribute the report by sending it to external platforms using the command `edr monitor send-report`.  
+Each specified platform in the command will be sent a report.  
```

Comment:
"You can specify more than one platform in the command, and the report will be sent to all."